server/main.ts

```diff
@@ -1,27 +1,36 @@
 import { NestFactory } from "@nestjs/core";
 import { AppModule } from "./app.module";
 import { join } from "path";
-import Logger from "./logger";
 import { NestExpressApplication } from "@nestjs/platform-express";
-import { ValidationPipe } from "@nestjs/common";
+import { ValidationPipe, Logger } from "@nestjs/common";
 import { SwaggerModule, DocumentBuilder } from "@nestjs/swagger";
-require("newrelic");
+import loadConfig from "./configLoader";
+import * as dotenv from "dotenv";
 const cookieParser = require("cookie-parser");
 const mongoose = require("mongoose");
+dotenv.config();
```

Comment:
thought (non-blocking): I believe there's a nodejs version that implements what dotenv does and this can be killed. We should find out which one and create an issue for that.

Reply:
#1132